### DISCUSSION TITLE
Anyway to adjust the number of previews for auto converter?

### DISCUSSION THREAD

**Discussion by Kleinja** [jlesage/docker-handbrake#387] [Q&A] [answered]
Super odd question, not sure if there currently is a solution for it. In the Handbrake GUI settings, there is a selection box for the number of previews (default is 10). Is there any way to adjust this value for the automatic video converter?

Here's my issue:
With one of my automatic conversions, I noticed some cropping was happening to the outer sides of the video. If I opened the source manually, the final dimensions in the previewer changed from the source dimensions. Same profile selected in the manual GUI as the AVC runs. 
Source: 
   Storage Size: 1918 x 802
   Display Size: 1918 x 802
   Aspect Ratio: 2.39:1

Final:
   Storage Size: 1522 x 792
   Display Size: 1522 x 792
   Aspect Ratio: 1.92:1

Turning off cropping matches the final resolution to the source.  Though I read somewhere that adjusting the number of previews has some sort of impact on this. So I bumped up the number of previews from 10 to 30 for testing, and reloaded the same file. Upon doing that, with cropping set to automatic the final dimensions matched the source. 

Manually running the process, the output file was no longer cropped and looked like I expected it to. Though, with the number of previews still set to 30, running it through the automatic converter still manages to crop the file. Seems this setting only changes the manual converter, not the auto converter as well.

I've had pretty good luck using just one main profile for converting media mostly autonomously. I haven't noticed any other media coming through like this before, except this one file. Might not have noticed except in the intro credits names were cutoff along the edges lol. 

Would be nice to have this as an environment variable to manually adjust, or if it is just hardcoded in, just have it increased to hopefully avoid this potential issue. Please let me know if I am missing something obvious. I know I left out a lot of information about the profile, but this is the easiest way to replicate said issue. I'd be happy to answer questions about the issue, or try some things if needed, thanks!

Image of the Number of Previews selection for reference: 
![image](https://github.com/user-attachments/assets/be44a1b8-c198-4c4d-8e1b-84932b49d581)

Dimensions tab with number of previews set to 10
![image](https://github.com/user-attachments/assets/bd725ac1-941d-4db8-9916-821c113c62f5)

Dimensions tab with number of previews set to 30
![image](https://github.com/user-attachments/assets/785923d5-1476-475a-9089-40918a71d035)

**Comment by jlesage** [comment, marked as answer]
You could try to set the `AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS` environment variable to `--previews 30`.

**Reply by Kleinja** [reply]
Thanks! I'll try that out when I have some free time. I didn't realize that was an environment option

**Reply by jlesage** [reply]
Great, let me know if this fixes your issue!
